concepticondata/conceptlists/Hale-1973-1798.tsv

```diff
@@ -1326,10 +1326,10 @@ Hale-1973-1798-1324	Human_Mind_and_Conditions-212	worship idols	v	10b3.22	257	WO
 Hale-1973-1798-1325	Human_Mind_and_Conditions-213	pray	v	10b3.23	24	PRAY
 Hale-1973-1798-1326	Human_Mind_and_Conditions-214	believe	v	10b3.24	1890	BELIEVE
 Hale-1973-1798-1327	Human_Mind_and_Conditions-215	offer to gods	v	10b3.25		
-Hale-1973-1798-1328	Human_Mind_and_Conditions-216	sacrifice	v	10b3.26		
+Hale-1973-1798-1328	Human_Mind_and_Conditions-216	sacrifice	v	10b3.26	1103	SACRIFICE
 Hale-1973-1798-1329	Human_Mind_and_Conditions-217	kill sacrificial animal	v	10b3.27		
 Hale-1973-1798-1330	Human_Mind_and_Conditions-218	bless	v	10b3.28	391	BLESS
-Hale-1973-1798-1331	Human_Mind_and_Conditions-219	sin	v	10b3.29		
+Hale-1973-1798-1331	Human_Mind_and_Conditions-219	sin	v	10b3.29	2971	SIN
```

Comment:
Would unmap, since "sin" is a noun in concepticon: https://github.com/concepticon/concepticon-data/blob/master/concepticondata/concepticon.tsv#L2967